src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -42,26 +37,10 @@ public override bool Execute()
                 {
```

Comment:
If we don't make ItemsToPush required again then we should handle the null/empty case instead of throwing. FWIW I think we should make it required.